torch/lib/TH/generic/THTensorRandom.c

```diff
@@ -160,18 +160,19 @@ void THTensor_(multinomialAliasSetup)(THTensor *probs, THLongTensor *J, THTensor
 }
 void THTensor_(multinomialAliasDraw)(THLongTensor *self, THGenerator *_generator, THLongTensor *J, THTensor *q)
 {
-  long K = THLongTensor_nElement(J);
-  long output_nelem = THLongTensor_nElement(self);
-  
-  int i = 0, _mask=0;
+  int64_t K = THLongTensor_nElement(J);
+  int64_t output_nelem = THLongTensor_nElement(self);
+  int64_t i = 0, _mask=0;
```

Comment:
Maybe this removes compiler warnings because of manipulations with the result of other operations, which are `int64_t`?